CHANGES.md

```diff
@@ -3,6 +3,7 @@
 - [REMOVED] Removed oneway methods from ReplicatorFactory. Users should 
   use the method described in the [replication documentation](https://github.com/cloudant/sync-android/blob/master/doc/replication.md) 
   instead
+- [NEW] Added query text search support.  Users can now create a text index and use the `{ "$text" : { "$search" : "..." } }` clause to perform full text search queries.
```

Comment:
To avoid any confusion/questions from the "example" here, perhaps you could just refer to the doc/query.md instead?At the very least, should provide a link to the query doc. You can also link to an anchor like so:

In doc/query.md
> #### Managing a Text Index <a name="managing-a-text-index"?></a>

In CHANGES.md
> Added query text search support. (See documentation)[doc/query.md#managing-a-text-index].